common/source/docs/common-radio-control-calibration.rst

```diff
@@ -1,28 +1,85 @@
-============================================
+=========================
+Radio Control Calibration
+=========================
 
-This article shows how to perform radio control calibration using
-*Mission Planner*.
+This article shows how to perform radio control calibration using Mission Planner
 
 Overview
 ========
 
-RC transmitters are used to control vehicle movement and orientation.
-Copter and Plane minimally control throttle, pitch, roll and yaw, while
-on Rover we just control throttle and roll. Each of these control
-signals are mapped to transmitter stick/switch(s) and in turn to
-autopilot channels from the connected receiver.
+RC transmitters allow the pilot to set the flight mode, control the vehicle's movement and orientation and also turn on/off auxiliary functions (i.e. raising and lowering landing gear, etc).
 
-Calibrating each of the transmitter controls/channels is a
-straightforward process - simply move each of the enabled
-sticks/switches through their full range and record the maximum and
-minimum positions.
+RC Calibration involves capturing each RC input channel's minimum, maximum and "trim" values so that ArduPilot can correctly interpret the input.
 
-Transmitter configuration
--------------------------
+Check the Transmitter's Setup
+-----------------------------
+
+- Ensure the battery is disconnected (this is important because it is possible to accidentally arm the vehicle during the RC calibration process)
+- Ensure the RC receiver is connected to the flight controller
+- Turn on your RC transmitter and if it has "trim tabs" ensure they are in the middle
+- Connect the flight controller to the PC using a USB cable
+- On the Mission Planner press the "Connect" button and open Mission Planner's **INITIAL SETUP \| Mandatory Hardware \| Radio Calibration** screen
+- Some green bars should appear showing the ArduPilot is receiving input from the Transmitter/Receiver. If no bars appear check the receiver's LED:
+
+   -  No lights may indicate that it is incorrectly wired to the autopilot.  Look for connectors that may have been inserted upside down
+   -  A Red or flashing LED may indicate that your RC transmitter/receiver need be bound.  See the manual that came with your RC equipment for instructions
+
+.. figure:: ../../../images/mp_radio_calibration.png
+   :target: ../_images/mp_radio_calibration.png
+
+- Check the channel mapping in the transmitter (ie. check which inputs channels are controlled by the transmitter's sticks, switches and knobs) by moving the sticks, knobs and switches and observing which (if any) green bars move.  If this is the first time the transmitter is being used with ArduPilot it is likely that the transmitter's channel mapping will need to be changed and normally this is done on the transmitter itself using its built-in configuration menu
+
+  - Determine if your transmitter is Mode1 or Mode2 (see below)
+  - Roll stick should control channel 1
```

Comment:
better put a picture for mode 1/ mode 2

Reply:
hi @khancyr, I've got one down at the bottom of the page.. I could make it more clear perhaps that the user should look down below in the "Mode1 vs Mode2" section.